css/tabBar.css

```diff
@@ -305,12 +323,9 @@ body.windows.dark-theme .navbar-action-button:not(:disabled):hover {
 /* in Carbon, close icon is a bit smaller than other icons, enlarge it in order to match with other icons */
 .tab-item .tab-close-button {
   font-size: 1.25em;
-  padding: 0.3em 0.45em;
-  margin-top: 0.15em;
+  box-sizing: border-box;
```

Comment:
Should the close icon not be the icon most to the right side?

![image](https://github.com/minbrowser/min/assets/31519867/b4fa9061-e719-4c66-b861-0c042685153e)

```suggestion
  box-sizing: border-box;
  order: 1;
```

Reply:
Yes it should.